### ISSUE TITLE
`terminus-font` uses a direct mirror link which no longer works

### ISSUE THREAD

**Issue by LoganDark** [kiss-community/community#1159]
```patch
--- a/community/terminus-font/sources
+++ b/community/terminus-font/sources
@@ -1 +1 @@
-https://deac-ams.dl.sourceforge.net/project/terminus-font/terminus-font-4.49/terminus-font-4.49.1.tar.gz
+https://downloads.sourceforge.net/project/terminus-font/terminus-font-MAJOR.MINOR/terminus-font-VERSION.tar.gz
--- a/community/terminus-font/version
+++ b/community/terminus-font/version
@@ -1 +1 @@
-4.49.1 1
+4.49.1 2
```

Here you go.

(I love `kiss-fork` so much)

**Comment by ioraff** [comment]
@aaronNGi

**Comment by LoganDark** [comment]
It looks like there are a lot of orphaned or otherwise unmaintained packages in this repo - that GitHub user hasn't been seen for over half a year. Maybe if I can get my trackpad and GPU to work under GKISS, I'd be open to adopting some.

**Comment by ioraff** [comment]
fixed with a1eb6b3

**Comment by LoganDark** [comment]
That works :)
